### DISCUSSION TITLE
Welcome to xmigrate Discussions!

### DISCUSSION THREAD

**Discussion by iamvishnuks** [vishnu-ks/Cloud-migration#41] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BishnoiK** [comment]
Hi @iamvishnuks - we are exploring migrate options , so came across this tool .. i am able to setup the tool which is good but while doing discovery of VM on AWS ubuntu machine as i am using pem file we are getting below error :

/app/ansible/XMigrateKey.pem' are too open.\r\nIt is required that your private key files are NOT accessible by others.\r\nThis private key will be ignored.\r\nLoad key \"/app/ansible/XMigrateKey.pem\": bad permissions\r\nubuntu@3.111.197.187: Permission denied (publickey).", "unreachable": true}

Can you help , just to be clear have changed the code a bit to include private key file as AWS VM is not getting authenticated via username and password. 

Above error is coming only via ansible i.e. via SSH i am able to login into VM on AWS using private key.
have provided read only access to pem file (suggest in AWS document - chmod 400 <keyfilename>) but still issue still persists during discovery process via ansible. Can you please help here if you have came across this issue.

I am  Windows machine -> running application on docker desktop Linux containers.

**Reply by iamvishnuks** [reply]
Hi Bishnoi this project will be soon going to be archived. We won't be doing any further updates on this project in favor of the new SaaS version of Xmigrate. You can find more details here, https://xmigrate.cloud/
If you would like to use Xmigrate SaaS tool, drop a mail to vishnu@xmigrate.cloud. We can setup a demo and take it forward. 
Thanks for showing interest in Xmigrate.
